core/src/main/java/org/kiwix/kiwixmobile/core/search/SearchFragment.kt

```diff
@@ -280,8 +291,14 @@ class SearchFragment : BaseFragment() {
       // `cancelAndJoin` cancels the previous running job and waits for it to completely cancel.
       renderingJob?.cancelAndJoin()
       isDataLoading = false
-      searchInTextMenuItem?.isVisible = state.searchOrigin == FromWebView
-      searchInTextMenuItem?.isEnabled = state.searchTerm.isNotBlank()
+      searchInTextMenuItem?.actionView?.isVisible = state.searchOrigin == FromWebView
```

Comment:
We are setting the alpha, enable, and disable at 3 places we should create a function and use that function to set these things.

Reply:
I have made the required changes, please confirm.